numpy/fft/tests/test_pocketfft.py

```diff
@@ -381,15 +389,6 @@ def test_all_1d_norm_preserving(self):
                     assert_allclose(x_norm,
                                     np.linalg.norm(tmp), atol=1e-6)
 
-    @pytest.mark.parametrize("dtype", [np.half, np.single, np.double,
```

Comment:
This one is now done much more thoroughly in `test_identity_long_short` and `test_identity_long_short_reversed`